docker-compose.yml

```diff
@@ -85,3 +85,22 @@ services:
     environment:
       - ENIGMA_ENV
       - SGX_MODE
+
+  mongo:
+    container_name: mongo
+    hostname: mongo
+    image: mongo
+    ports:
+      - "27017:27017"
```

Comment:
Do you need this as `ports` (mapped to host) rather than just `expose` (just for docker)?

Reply:
You're probably right. i copied this from `enigmampc/salad where the containers communicate through the host, and this was necessary.